recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "arcticdb" %}
-{% set version = "2.0.0" %}
+{% set version = "3.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 149660905f2a1169a8bfeeb0f58e35e496747589a7d2147252181c4e826b3683
+  sha256: 6d2d68e8b931d8a871f9034eaabe48acdfe3578d7c5862122dce119b7d867947
 
   patches:
```

Comment:
Side-note: the `patches` section can be removed since https://github.com/man-group/ArcticDB/issues/662 has been merged.